src/components/button.jsx

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+
+class Button extends Component {
+
+    
+
+    render(){
+        return(
+            <>
+            <button onClick={this.handleClick}>Button</button>
```

Comment:
Where is the click handler?
You should be able to create a custom hook for a click handler in the helper folder to import here, or you can create one in the page which will call these components and pass it down as a prop.

Reply:
Added click handler:
`import React, {Component} from 'react'

class Button extends Component {

    state = {
        submit: false
    }

    handleClick = (e) => {
        const clicked = this.state.submit
        this.setState({
            submit: !clicked
        })
    }

    render(){
        return(
            <>
            <button onClick={this.handleClick}>{this.state.submit ? "Saved" : "Save?"}</button>
            </>
        )

    }
}

export default Button
`